doc-content/jbpm-docs/src/main/asciidoc/DesignBuildCases/case-management-stages-con.adoc

```diff
@@ -1,12 +1,12 @@
 [id='case-management-stages-con-{context}']
 = Stages
 
-Case management stages are a collection of tasks. A stage is an ad hoc sub-process that can be defined using the process designer and may include other case management nodes, such as a milestone. A milestone can also be configured as completed when a stage or a number of stages are completed. Therefore, a milestone may be activated or achieved by the completion of a stage, and a stage may include a milestone or a number of milestones.
+Case management stages are a collection of tasks. A stage is an ad hoc sub-process that can be defined using the process designer and can include other case management nodes, such as a milestone. A milestone can also be configured as completed when a stage or a number of stages are completed. Therefore, a milestone may be activated or achieved by the completion of a stage, and a stage may include a milestone or a number of milestones.
```

Comment:
"may be activated/may include" breaks the style guide

Reply:
And "a milestone can be configured" is a rather bad case of passive voice, in my view